appveyor.yml

```diff
@@ -0,0 +1,32 @@
+version: '{build}'
+os: Visual Studio 2019
+branches:
+  only:
+  - visual_studio_build # TODO change to master 
+
+matrix:
+  fast_finish: true
+
+platform:
+  - x64
+
+configuration:
+  - Release
+
+install:
+- set PATH=%PATH%;c:\Program Files (x86)\Microsoft Visual Studio 15.0\Common7\IDE\
+- cd %APPVEYOR_BUILD_FOLDER%
+- git submodule update --init
+- mkdir build
+- cd build%
+
+build_script:
+- cmake -G "Visual Studio 16 2019" -A x64 -DPYTHON_EXECUTABLE="C:/Python36-x64/python.exe" -DCMAKE_INSTALL_PREFIX=../install ..
+- cmake --build . --config %CONFIGURATION% --target install all-tests
+
+test_script:
+- ctest -VV 
```

Comment:
I think we should probably not be using `ctest` to run tests. As far as I remember it had trouble picking up some templated tests, i.e., not all tests were being run.

Reply:
Unfortunately I think you are right different total test count between appveyor and travis. I'll change this.